readme.md

```diff
@@ -0,0 +1,14 @@
+# Prerequisites
+- conan
+- cmake
+- C++17 compliant compiler
+
+# Build
+```bash
+$ mkdir build && cd build
+$ conan remote add d34dpkgs https://api.bintray.com/conan/0xd34d10cc/d34dpkgs
+$ conan remote add bincrafters https://api.bintray.com/conan/bincrafters/public-conan
+$ conan install .. --build=missing
+$ cmake ..
```

Comment:
Doesn't windows setup require to select generator?

Reply:
It is required only in case if several VS instances are installed